lib/plugins/aws/deploy/lib/iam-policy-lambda-execution-template.json

```diff
@@ -1,6 +1,9 @@
 {
   "IamPolicyLambdaExecution": {
     "Type": "AWS::IAM::Policy",
+    "DependsOn": [
+      "IamRoleLambdaExecution"
```

Comment:
@pmuens Please wait, I'm currently checking if this is always the case and no changes are preformed after including this.